.ci/scripts/clang-format.sh

```diff
@@ -20,9 +20,11 @@ echo "Starting format check..."
 
 RETURN_CODE=0
 
+CLANG_FORMAT_EXE=${CLANG_FORMAT_EXE:-clang-format-14}
```

Comment:
Since specific version is going to be used, clang-format alias setup should be removed from `.ci/env/apt.sh`:
https://github.com/oneapi-src/oneDAL/blob/2b18f4d4e1f3c3851c6af7b42d59d4b388728e40/.ci/env/apt.sh#L43-L47

Reply:
This is used by the CI right? It is installing a specific version which allows us to set the default version to this.

Reply:
Last two lines set generic `clang-format` alias which should be removed for consistency since specific version is going to be used.

Reply:
done.